src/components/document/tile-row.tsx

```diff
@@ -70,12 +71,15 @@ export class TileRowComponent extends BaseComponent<IProps, IState> {
 
   public render() {
     const { model } = this.props;
+    const { isSectionHeader, sectionId } = model;
     const height = this.props.height || model.height;
     const style = height ? { height } : undefined;
     return (
       <div className={`tile-row`} data-row-id={model.id}
           style={style} ref={elt => this.tileRowDiv = elt}>
-        {this.renderTiles(height)}
+        {isSectionHeader && sectionId ?
+          <SectionHeader type={sectionId}/>
+          : this.renderTiles(height)}
         {!this.props.readOnly && this.renderDragDropHandles()}
```

Comment:
```suggestion
        {isSectionHeader && sectionId && <SectionHeader type={sectionId}/>}
        {!isSectionHeader && this.renderTiles(height)}
        {!isSectionHeader && !this.props.readOnly && this.renderDragDropHandles()}
```
I think we don't want to render the drag-drop handles for the section header.